app.rb

```diff
@@ -0,0 +1,103 @@
+require './arcade_main'
+require './main_helper'
```

Comment:
### Optional
- [ ] It would be nice if we use `require_relative` for files in our current directory :+1: